config.py

```diff
@@ -89,31 +89,47 @@
 
 DIVERT_TO_PRIVATE = ('help', )
 
-ROOMS_TO_JOIN = (
```

Comment:
It would be better to have all rooms which are common stay in the variable at the top here, ..

but change it from a tuple `(..)` to a list `[ ... ]`

so this would be

```py
ROOMS_TO_JOIN = [
```

Reply:
Got it.